src/components/Datatable/ToolsTabItem/ToolsTabItem.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { isNotUndefined, noop } from 'ramda-adjunct';
+import styled from 'styled-components';
+
+import { SpacingSizeValuePropType } from '../../../types/spacing.types';
+import { getColor, pxToRem } from '../../../utils/helpers';
+import { IconTypes, SSCIconNames } from '../../Icon/Icon.enums';
+import { SSCIcons, Types } from '../../Icon/Icon.types';
+import Button from '../../Button/Button';
+import StyledIcon from '../../Button/StyledIcon';
+import { ButtonSizes } from '../../Button/Button.enums';
+import { ToolsTabItemProps } from './ToolsTabItem.types';
+import { ToolsTabItemStates } from './ToolsTabItem.enum';
+
+const DefaultToolsTabItem = styled(Button)`
+  &&& {
+    padding: ${pxToRem(9, 12)};
+    color: ${getColor('graphite2B')};
+    &:focus:not(:disabled),
+    &:hover:not(:disabled),
+    &:not([href]):not([tabindex]):not(:disabled):hover,
+    &.hover,
+    &.focus {
+      color: ${getColor('graphite4B')};
+      svg {
+        color: ${getColor('graphite2B')};
+      }
+    }
+  }
+`;
+
+const ActiveToolsTabItem = styled(Button)`
+  &&& {
+    padding: ${pxToRem(6, 12)};
+    background-color: ${getColor('graphite5H')};
+    color: ${getColor('blueberryClassic')};
+    border-radius: ${pxToRem(20)};
+    border-color: ${getColor('blueberryClassic')};
+    border-width: 2px;
+    border-style: solid;
+
+    &:focus:not(:disabled),
+    &:hover:not(:disabled),
+    &:not([href]):not([tabindex]):not(:disabled):hover,
+    &.hover,
+    &.focus {
+      background-color: ${getColor('graphite5H')};
+      color: ${getColor('radiantBlueberry')};
+      border-color: ${getColor('radiantBlueberry')};
+    }
+  }
+`;
+
+const AppliedToolsTabItem = styled(Button)`
+  &&& {
+    padding: ${pxToRem(9, 12)};
+    color: ${getColor('graphite2B')};
+
+    svg {
+      color: ${getColor('blueberryClassic')};
+    }
+
+    &:focus:not(:disabled),
+    &:hover:not(:disabled),
+    &:not([href]):not([tabindex]):not(:disabled):hover,
+    &.hover,
+    &.focus {
+      color: ${getColor('graphite4B')};
+      svg {
+        color: ${getColor('radiantBlueberry')};
+      }
+    }
+  }
+`;
+
+const ToolsTabItem: React.FC<
+  ToolsTabItemProps & React.ComponentProps<typeof Button>
+> = ({
+  children,
+  itemState = ToolsTabItemStates.default,
+  activateTool = noop,
+  deactivateTool = noop,
+  iconName,
+  iconType = IconTypes.ssc,
+  margin = 'none',
+  isDisabled = false,
+  ...props
+}) => {
+  const [currentItemState, setCurrentItemState] = useState(itemState);
+
+  const commonProps = {
+    size: ButtonSizes.sm,
+    variant: 'text',
+    margin,
+    ...props,
+  };
+
+  const onClickDefault = () => {
+    activateTool();
+    setCurrentItemState(ToolsTabItemStates.active);
+  };
+
+  const onClickActive = () => {
+    deactivateTool();
+    setCurrentItemState(ToolsTabItemStates.applied);
+  };
+
+  const onClickApplied = () => {
```

Comment:
this is the same as `onClickDefault`

Reply:
changed to `onClickActive` and `onClickNotActive`